Sheikh_Shahzeb/assignment_wizard/ninja.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'human'
+class Ninja < human
+	def initialize 
+		@stealth = 175
+		self
+	end
```

Comment:
you still need health, intelligence, and strength from the initialize method in the human class that you're inheriting from.  The way it's written you're ninja initialize is overwriting the human initialize.  This is where you need 'super' so that you're still using the human class initialize method but still able to change what you want